data/json/monsters/insect_spider.json

```diff
@@ -1857,5 +1857,347 @@
+    "melee_dice_sides": 6,
+    "melee_cut": 12,
+    "dodge": 3,
+    "armor_bash": 10,
+    "armor_cut": 16,
+    "armor_bullet": 13,
+    "vision_day": 15,
+    "vision_night": 5,
+    "harvest": "arachnid",
+    "anger_triggers": [ "PLAYER_WEAK", "PLAYER_CLOSE"],
+    "fear_triggers": [ "FIRE" ],
+    "death_function": [ "NORMAL" ],
+    "special_attacks": [  [ "RANGED_PULL", 30 ], [ "GRAB_DRAG", 10 ], { "type": "leap", "cooldown": 8, "max_range": 5, "allow_no_target": false }],
+    "flags": ["POISON", "SEES", "HEARS", "SMELLS", "CLIMBS", "PATH_AVOID_FIRE","PATH_AVOID_FALL", "STUMBLES" ]
+  }, 
+  {
+    "id": "mon_mantis_mega",
+    "type": "MONSTER",
+    "name": { "str": "great mantis" },
+    "description": "This ferocious creature is lurking through the land.  It's probably capable of killing a moose with its powerful arms.",
+    "copy-from": "mon_mantis_giant",
+    "proportional": { "hp": 20, "speed": 0.25, "vision_day": 2 },
+    "volume": "625 L",
+    "symbol": "M",
+    "color": "red_green",
+    "weight": "815 kg",
+    "melee_dice": 2,
+    "melee_cut": 20,
+    "dodge": 2,
+    "special_attacks": [
+      {
+        "type": "bite",
+        "cooldown": 4,
+        "accuracy": 5,
+        "damage_max_instance": [ { "damage_type": "stab", "amount": 15, "armor_multiplier": 0.6 } ]
+      }
+    ],
+    "extend": { "flags": [ "DESTROYS", "PUSH_MON", "PUSH_VEH" ] },
+  "delete": { "type": "leap", "cooldown": 8, "max_range": 5, "allow_no_target": false }
+  },
+ {
+    "id": "mon_mole_cricket",
+    "type": "MONSTER",
+    "name": { "str": "giant mole cricket" },
+    "description": "An enormous underground bug.  It's a common garden pest.  Beware ear-splitting screech and strong shovel-like front legs.",
+    "default_faction": "insect",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "150 L",
+    "weight": "200 kg",
+    "hp": 160,
+    "speed": 80,
+    "material": [ "iflesh" ],
+    "symbol": "M",
+    "color": "brown",
+    "aggression": -10,
+    "morale": 60,
+    "melee_skill": 6,
+    "melee_dice": 4,
+    "melee_dice_sides": 6,
+    "melee_cut": 10,
+    "armor_bash": 15,
+    "armor_cut": 18,
+    "armor_bullet": 20,
+    "vision_day": 10,
+    "vision_night": 15,
+    "harvest": "arachnid",
+    "special_attacks": [ [ "SHRIEK_ALERT", 10 ], [ "SHRIEK_STUN", 5 ], [ "impale", 15 ] ],
+    "anger_triggers": [ "HURT", "SOUND", "STALK" ],
+    "fear_triggers": [ "FIRE" ],
+    "death_function": [ "NORMAL" ],
+    "flags": [ "POISON","ATTACKMON", "BORES", "CAN_DIG", "HEARS", "KEENNOSE", "PATH_AVOID_DANGER_1", "SMELLS", "WARM" ],
+    "//": "Reinsert GOODHEARING when z-level tunneling is possible."
+  },
+    {
+    "id": "mon_lady_bug_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant lady bug" },
+    "description": "A red lady bug covered in black spots about a size of a horse.  Despite cute looks it is a ferocious predator capable of killing swarms of ants and aphids.",
```

Comment:
```suggestion
    "description": "A lady beetle roughly the size of a horse.  Its forewings are red with black spots, a distinctive appearance traditionally associated with good luck and adored by children.  Despite cute looks it is a ferocious predator capable of killing swarms of ants and aphids.",
```

Reply:
If you want to emphasize it's a beetle, you you should still keep its name (i.e. ladybug beetle). I'd probably not use a description of the normal sized ones, though, as I don't think this monster would be associated with either good luck or be adored by children. If you somehow would want to keep it, I'd change it to something along the lines of "This monster was cute with its black spotted red forewings and was adored by children as well as associated with good luck when it was the normal size, but this is clearly a ferocious predator..."